src/platform-includes/performance/opentelemetry-setup/node.mdx

```diff
@@ -17,25 +25,71 @@ const {
 const {
   OTLPTraceExporter,
 } = require("@opentelemetry/exporter-trace-otlp-grpc");
+const {
+  AsyncLocalStorageContextManager,
+} = require("@opentelemetry/context-async-hooks");
 
-// Make sure to call `Sentry.init` BEFORE initializing the OpenTelemetry SDK
-Sentry.init({
-  dsn: "___PUBLIC_DSN___",
-  tracesSampleRate: 1.0,
-  // set the instrumenter to use OpenTelemetry instead of Sentry
-  instrumenter: "otel",
-  // ...
-});
+function setupSentry() {
+  setupGlobalHub();
+
+  // Make sure to call `Sentry.init` BEFORE initializing the OpenTelemetry SDK
+  Sentry.init({
+    dsn: "___PUBLIC_DSN___",
+    tracesSampleRate: 1.0,
+    // set the instrumenter to use OpenTelemetry instead of Sentry
+    instrumenter: "otel",
+    // ...
+  });
+
+  const client = getCurrentHub().getClient();
+  setupEventContextTrace(client);
+
+  // You can wrap whatever local storage context manager you want to use
+  const SentryContextManager = wrapContextManagerClass(
+    AsyncLocalStorageContextManager
+  );
+
+  const sdk = new opentelemetry.NodeSDK({
+    // Existing config
+    traceExporter: new OTLPTraceExporter(),
+    instrumentations: [getNodeAutoInstrumentations()],
+
+    // Sentry config
+    spanProcessor: new SentrySpanProcessor(),
+    textMapPropagator: new SentryPropagator(),
+    contextManager: new SentryContextManager(),
+    sampler: new SentrySampler(client),
+  });
 
-const sdk = new opentelemetry.NodeSDK({
-  // Existing config
-  traceExporter: new OTLPTraceExporter(),
-  instrumentations: [getNodeAutoInstrumentations()],
+  // Ensure OpenTelemetry Context & Sentry Hub/Scope is synced
+  setOpenTelemetryContextAsyncContextStrategy();
 
-  // Sentry config
-  spanProcessor: new SentrySpanProcessor(),
-  textMapPropagator: new SentryPropagator(),
+  sdk.start();
+}
+
+setupSentry();
+```
+
+Note that with this setup, you can _only_ use OpenTelemetry tracing for performance. `Sentry.startTransaction()` will _not_ work.
```

Comment:
@shanamatthews I updated this actually and made this more explicit. I specifically call out `startTransaction()` as not working (as that is really the main thing that will not work) but also explain what to do instead!

Reply:
This looks good to me!